modules/jupyter-widget/src/playground/playground.js

```diff
@@ -90,26 +91,33 @@ export function processDataBuffer({binary, convertedJson}) {
   return convertedJson;
 }
 
+// Handles a general event
+function sendEventViaTransport(transport, name, data) {
+  transport.jupyterModel.set('deck_event', JSON.stringify({name, data}));
```

Comment:
Expected to be available within the Python-side of the widget

An observer could be set here, which could trigger a Python function

Reply:
To-do: Figure out how a rapid series of events, like many view state changes, would be received by Python